app/both/collections/residencies.js

```diff
@@ -40,6 +40,14 @@ var ResidenciesSchema = new SimpleSchema({
         }
       }
     },
+    custom() {
+      const moveOut = this.value
+      const moveIn = this.field('moveIn').value
+      const result = validateDate(moveIn, moveOut)
+      if (result !== true) {
+        return "badDate"
```

Comment:
Should this say "bad move out date"?

Reply:
it is an autoform code, based on it it will render an error message saying 'Move out date is invalid'